src/Std/Time/Zoned.lean

```diff
@@ -17,7 +17,7 @@ set_option linter.all true
 namespace PlainDateTime
 
 /--
-Get the current time.
+Get the current time, in the current timezone.
```

Comment:
Elsewhere we use the term "local" instead of "current" to refer to the time zone set on the system that the code is running on.

As you suggested, I think it would be helpful to include something like "To obtain the current time in a specific timezone, use `DateTime.now` or `ZonedDateTime.nowAt`."

Reply:
Good points! I have addressed both of them. I only included such a sentence once, as I didn't see how to apply the same to the `date` or `time` methods.